rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1989,8 +1989,7 @@ def test_upgrade_db_40_to_41(user_data_dir):
     )""") is True
         cursor.execute('INSERT INTO key_value_cache VALUES (?, ?)', ('test', 'test'))
         cursor.execute('SELECT * FROM key_value_cache')
-        assert cursor.fetchone() == ('test', 'test')
-        assert cursor.fetchall() == []
+        assert ('test', 'test') in cursor.fetchall()
```

Comment:
```suggestion
        assert [('test', 'test')] == cursor.fetchall()
```

Reply:
You should add to the test db values for all the settings and check that they got correctly moved